helpers/content/team.js

```diff
@@ -0,0 +1,1033 @@
+				pictureUrl: '/images/team/Marco_Schaarschmidt.jpg',
+				priority: 0,
+				social: {
+					facebook: '',
+					twitter: '',
+					linkedin: '',
+				},
+				longDesc: false,
+			},
+			{
+				title: '',
+				firstName: 'Sebastian',
+				lastName: 'Serth',
+				position: 'Ehem. Masterstudent',
+				description: 'Sebastian hat seine Masterarbeit im Kontext der HPI Schul-Cloud geschrieben. Als Grundlage dazu dienten ihm Erkenntnisse von openHPI, der Online-Lernplattform des HPI, in dessen Rahmen er bereits seine Bachelorarbeit angefertigt hat. In seiner Masterarbeit untersuchte er, wie Lehrer individuelle Arbeitsblätter mit interaktiven Programmieraufgaben erstellen und im Unterricht einsetzen können: die Schüler arbeiten dazu wie bei Online-Kursen in der webbasierten Programmierumgebung CodeOcean, in der sie ihre Programme entwickeln, ausführen und mit Hilfe automatisierter Tests ihrer Lehrer sogar eigenständig überprüfen können. Die Lehrer erhalten einen besseren Überblick über den Lernfortschritt der Klasse und können so bei auftretenden Problemen zeitnah und zielorientiert Unterstützung leisten. In seiner Freizeit springt Sebastian leidenschaftlich gern auf dem Großtrampolin, reist mit Vergnügen durch fremde Länder und hält dabei seine Eindrücke fotografisch fest.',
+				pictureUrl: '/images/team/Sebastian_Serth.jpg',
+				priority: 0,
+				social: {
+					facebook: '',
+					twitter: '',
+					linkedin: '',
+				},
+				longDesc: false,
+			},
+			{
+				title: '',
+				firstName: 'Johannes',
+				lastName: 'Unruh',
+				position: 'Ehem. Bachelorstudent',
+				description: 'Johannes studierte am Hasso-Plattner-Institut IT-Systems Engineering im Bachelor und war Teil des Bachelorprojektes 2018/19 bei der HPI Schul-Cloud. Zuvor hat er einen Master in Wirtschaftsingenieurwesen an der TU Berlin erworben. Neben der Uni beschäftigt er sich mit Themen der Künstlichen Intelligenz und des Web Development.',
+				pictureUrl: '/images/team/Johannes_Unruh.jpg',
+				priority: 0,
+				social: {
+					facebook: '',
+					twitter: '',
+					linkedin: '',
+				},
+				longDesc: false,
+			},
+			{
+				title: '',
+				firstName: 'Florian',
+				lastName: 'Wirtz',
+				position: 'Ehem. Bachelorstudent',
+				description: 'Florian war studentische Hilfskraft im Team HPI Schul-Cloud sowie Mitglied des Bachelorprojekts 2017/18. Seine Arbeit dort war vor allem durch Forschung im Bereich der Content-Erstellung und -Integration charakterisiert. In seiner Freizeit engagiert er sich ehrenamtlich in der Ausrichtung von Hackathons und findet Ausgleich im Radfahren, Sport und Reisen.',
+				pictureUrl: '/images/team/Florian_Wiertz.jpg',
+				priority: 0,
+				social: {
+					facebook: '',
+					twitter: '',
+					linkedin: '',
+				},
+				longDesc: false,
+			},
+		],
+	},
+].filter(t => t.team.length).map((t, i) => {
+	t.style = sectionTypes[i % sectionTypes.length];
+	return t;
+});
```

Comment:
And here we filter out sections without members and add the style (color) attribute